runtime/executor/src/main/java/edu/snu/onyx/runtime/executor/data/metadata/FileMetadata.java

```diff
@@ -71,14 +92,17 @@ protected FileMetadata(final boolean partitionCommitPerWrite) {
   public abstract void deleteMetadata() throws IOException;
```

Comment:
What do you think about replacing this with `Closeable#close`?

Reply:
Deleting metadata and closing metadata are different.
The metadata have to exist in file even after the metadata is closed.